docs/how-to-guides/otel-collector.md

```diff
@@ -20,6 +20,73 @@ Below we include a couple of examples for using the OpenTelemetry collector, ass
 This documentation does not attempt to be a complete guide to the OpenTelemetry collector, but rather a gentle introduction along with some key examples.
 For more information on the collector please see the [official documentation](https://opentelemetry.io/docs/collector/).
 
+## Sink data into AWS S3
+
+If you want to keep your data stored long-term, the OpenTelemetry Collector offers a great way to send it directly to AWS S3.
+You need to configure Logfire SDK to send data to the OpenTelemetry Collector, which will then forward the data to AWS S3.
+
+Here is a simple example of how to send data into Logfire dashboard and AWS S3.
+
+First, you'll need to set up two things in AWS:
+
+1. An S3 bucket.
+2. An IAM user who has permission to write to that S3 bucket.
+
+Here is the example OpenTelemetry Collector configuration that sends data to AWS S3:
+
+```yaml title="config.yaml"
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: "0.0.0.0:4318"
+exporters:
+  awss3:
+    s3uploader:
+      region: ${env:AWS_REGION}
+      s3_bucket: ${env:AWS_S3_BUCKET}
+processors:
+  batch:
+    timeout: 10s
+    send_batch_size: 32768
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [awss3]
+    logs:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [awss3]
+```
+
+Run the OpenTelemetry Collector:
+
+```shell
+docker run -v $(pwd)/config.yaml:/etc/otelcol-contrib/config.yaml -e AWS_REGION=<aws-region> -e AWS_S3_BUCKET=<bucket-name> -e AWS_ACCESS_KEY_ID=<aws-access-key-id> -e AWS_SECRET_ACCESS_KEY=<aws-secret-access-key> -p 4318:4318  otel/opentelemetry-collector-contrib
```

Comment:
```suggestion
docker run \
    -v ./config.yaml:/etc/otelcol-contrib/config.yaml \
    -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
    -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
    -p 4318:4318 \
    otel/opentelemetry-collector-contrib
```

- avoid horizontal scroll
- people often have aws creds in env vars already, so this might work as is
- i think rather put the region and bucket name in the config file for simplicity